cpp/fetcher/fetcher.cc

```diff
@@ -74,9 +88,11 @@ struct FetchState {
 
 
 FetchState::FetchState(Database<LoggedCertificate>* db,
-                       unique_ptr<PeerGroup>&& peer_group, Task* task)
+                       unique_ptr<PeerGroup>&& peer_group,
```

Comment:
You're talking about the `unique_ptr<PeerGroup>`? That one's yours  :)

I'd rather keep this PR about SCT/STH verification and do cleanup in another...